### ISSUE TITLE
Migrate WCA translations to the website translation system

### ISSUE THREAD

**Issue by lgarron** [thewca/worldcubeassociation.org#4536]
This has been suggested a few times, and I think it would help significantly to avoid outdated translations.

The tradeoff I see are:
- The bar to accepting translations might be higher (people can't just send us text documents).
  - Per-Regulation translation could lead to inconsistency over time. It would be useful if we could show related Regulations or show excepts from a core translations glossary to ensure the same term is translated consistently.
- We might need a way to handle half-translated Regulations
  - Can we show the English version and have an icon for "contribute a translation!"?
- We need a way to get translations before the English version is released.
  - A simple approach might be to have a "release candidate" page.

**Comment by hanwu85** [comment]
@lgarron  Is this what you are referring to?
https://www.worldcubeassociation.org/translations/status

**Comment by jfly** [comment]
@hanwu85 yes. There's more information about how WCA website translations work over on https://github.com/thewca/worldcubeassociation.org/wiki#are-you-interested-in-translating-the-website-into-your-native-language

**Comment by moralsh** [comment]
Are we talking about creating a tool similar to the what Jonatan did to help translate the website? 

That would be a really great Idea and I would be really interested in helping with it, we could even add some configuration to translate from a release candidate branch in English to a similar branch in the wca-regulations-translations repository.

I've been trying to get myself to update the severely outdated Spanish translation for quite some time but the fact that it's almost like starting from scratch kills my good intentions often, a tool would help lots.

My lack of experience with ruby prevents me from offering myself to do the job, but I can surely help once a skeleton of the app is done and have definitely have some ideas that could make translating the regulations easier.

If we are not talking about making a tool, scratch my whole message and just take it as an idea that could work.

**Comment by lgarron** [comment]
> Are we talking about creating a tool similar to the what Jonatan did to help translate the website?

Yeah, but I would strongly prefer to extend/adapt Jonatan's tool rather than writing a new one.

cc @jonatanklosko in case he has thoughts.
